Dockerfile.prod

```diff
@@ -1,51 +1,66 @@
-FROM node:20-alpine3.18 as build
-# Installing libvips-dev for sharp Compatibility
-RUN apk update && apk add --no-cache git libc6-compat build-base gcc bash autoconf automake zlib-dev libpng-dev vips-dev > /dev/null 2>&1
+FROM node:20-alpine3.18 AS build
+# Update package index and install build dependencies.
+# These packages are needed for building native modules and ensuring compatibility.
+RUN apk update && apk add --no-cache \
```

Comment:
Zou je hierboven niet ook `RUN set -eux` willen doen? Die kan misschien als losse eerste stap, dan delen alle images die layer.

Of blijft `-eux` alleen geldig voor die éne command?

Reply:
Ik heb `set -eux` toegevoegd omdat je direct moet afsluiten als een opdracht mislukt (handig voor het debuggen en om te zorgen dat builds niet doorgaan na fouten).